migrations/schemas/20220527140133-change-column-name-stickers.sql

```diff
@@ -0,0 +1,10 @@
+
+-- +migrate Up
+ALTER TABLE "discord_guild_stats"
+    RENAME COLUMN nr_of_guild_stickers TO nr_of_server_stickers;
+    
+ALTER TABLE "discord_guild_stats"
+    RENAME COLUMN nr_of_standard_stickers TO nr_of_custom_stickers;
+
+-- +migrate Down
+
```

Comment:
migrate down to add back old fields when rollback, ex:
- Migrate up: `ALTER TABLE "discord_guild_stats"
    RENAME COLUMN nr_of_guild_stickers TO nr_of_server_stickers;`
- Migrate down: `ALTER TABLE "discord_guild_stats"
    RENAME COLUMN nr_of_server_stickers TO nr_of_guild_stickers;`